mix_helpers.exs

```diff
@@ -136,6 +136,16 @@ defmodule Mix.Appsignal.Helper do
   end
   defp map_arch(_), do: :unsupported
 
+  defp tmp_dir do
+    default_tmp_dir = "/tmp"
+
+    case {File.dir?(default_tmp_dir), File.stat(default_tmp_dir)} do
+      {true, {:ok, %{access: :write}}} -> default_tmp_dir
+      {true, {:ok, %{access: :read_write}}} -> default_tmp_dir
+      _ -> System.tmp_dir
```

Comment:
This should be a `System.tmp_dir!` (with a bang). We don't have a solution to when everything fails, and using the bang will make sure this'd crash on this line, instead of returning `nil` and it crashing later (or writing to somewhere unexpected).